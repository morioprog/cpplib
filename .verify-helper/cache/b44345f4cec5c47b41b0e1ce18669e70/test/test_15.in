10
9 8
9 7
-3 2
-1 1
-8 9
2 9
3 2
6 5
1 1
8 1
