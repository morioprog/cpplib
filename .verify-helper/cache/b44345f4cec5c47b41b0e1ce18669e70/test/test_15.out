8 1
3 2
9 7
6 5
9 8
1 1
2 9
-8 9
-1 1
-3 2
